machine-translation/README.md

```diff
@@ -0,0 +1,48 @@
+* [MarianNMT](https://marian-nmt.github.io/) is a machine translation framework written in C++ with minimal dependencies. Its advantages are that it's *fast* and it comes with ready-to-use use deployment solutions. However, being written in C++ might make it harder to get started with it.
+
+Overall we recommend starting with FairSeq due to its relative ease of use. A good starting point is the [scripts for training small models in WMT14](https://github.com/pytorch/fairseq/tree/main/examples/translation). Another possible approach is finetuning a multilingual model pretrained on many languages, such as the [M2M-100 model](https://github.com/pytorch/fairseq/tree/main/examples/m2m_100), especially if such languages are similar to the languages we are targeting.
+
+## Evaluating a Machine Translation System
+
+Evaluating a machine translation system is very much an open problem and an active research area. The best way to evaluate such systems is to do a **human evaluation** since it's the most reliable way to compare how the translations provided by the model compare to human references. For example, one might hire annotators that speak both languages that our system targets from freelancing platforms such as [Upwork](https://www.upwork.com/) and asking them to rate a system, as well as the human reference translations
+
+However, human evaluation is expensive, making it hard to use as evaluation for iterating over models. Fortunately, automatic metrics exist. While they aren't as reliable as human evaluation, they are cheap to compute and generally allow us to tell if a model is improving. Some commonly used metrics are:
+
+* **BLEU**: Based on comparing lexical features of the reference and system translation, BLEU has been the *de-facto* standard for evaluating MT systems. While many concerns have been raised recently about its correlation with human judgements, it's still widely reported when evaluating MT systems, since it is fast to compute and well-studied.
+* **COMET**: Based on recent advances in pretrained cross-lingual language models, COMET is a neural evaluation metric that was trained to predict human quality assessments and it has been shown to correlate better human judgements than lexical-based metrics such as BLEU, therefore being a better indicator of a system's performance. However, unlike lexical metrics, it only supports a pre-defined set of languages and can be more expensive to compute than BLEU.
+
+The recommended way to evaluate a system is then to use an *automatic metric* to provide cheap evaluations of a system while iterating over approaches and, when confident that the model is good, performing a final *human evaluation* to have a more accurate assessment of its quality.
```

Comment:
I made this suggestion out of my own judgment @neubig , let me know if you disagree with it